selfdrive/ui/ui.cc

```diff
@@ -118,7 +118,9 @@ static void update_model(UIState *s, const cereal::ModelDataV2::Reader &model) {
 }
 
 static void update_sockets(UIState *s) {
-  s->sm->update(0);
+  // ensures UI stays responsive when modelV2 is not alive
+  int timeout = s->sm->alive("modelV2") ? 1000 / UI_FREQ : 0;
+  s->sm->update(timeout);
```

Comment:
Kept the QTimer at 50 ms, as that enforces a consistent rate still, then we wait anywhere from 3 to 7 ms on PC replay, and {} to {} ms on C3